.github/workflows/wheel_macOS.yml

```diff
@@ -0,0 +1,57 @@
+name: wheel - macOS
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_macos:
+    name: macOS wheel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        python-version: [3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: check C++ compiler and Python
+        run: |
```

Comment:
I think clang --version and python --version are enough. That's two lines instead of five.

Reply:
I can remove the other lines.

Usually, I might use some special compiler/python so printing the which is an old habit (you don't believe how often people run tests with the wrong python)